health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -472,71 +475,98 @@ const FormulaConfiguration = ({ onSelect, category, customProps, formulas: initi
             }}
             footerChildren={[
+              <Button
+                type={"button"}
+                size={"large"}
+                variation={"primary"}
+                label={t("NO")}
+                onClick={() => {
+                  setFormulasPopUp(false);
+                  setSelectedDeletedFormula(null);
+                }}
+              />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)